### ISSUE TITLE
[Feature] Chatbot for Signup form

### ISSUE THREAD

**Issue by satyammittal** [scalability4all/voice-enabled-chatbot#19]
**Is your feature request related to a problem? Please describe.**
A chatbot in place of Sign up forms are more interactive. Providing clients the way to register in interactive manner rather than with forms

**Describe the solution you'd like**
API and storage at backend. We can use a table storing which forms are required by client.

**Describe alternatives you've considered**
Api from the Google form or any other agent

**Additional context**
---

**Comment by Mishrasubha** [comment]
Hello, I would like to suggest to use GraphQL and SQLALchemy for API and storage at the backend.  GraphQL is better than REST and has many other advantages.  We can use the Graphene-Python library with supports building of GraphQL and storage with the use of SQLAlchemy. 
Would like to know your feedback on this.

**Comment by satyammittal** [comment]
@Mishrasubha Yes looks fine, you can start working on a small design like only username and email address and show how chatbot can covert a form to interactive manner.

**Comment by Mishrasubha** [comment]
Okay then I will start working on it
